### ISSUE TITLE
Auto-fixing 'no-get' with 'useOptionalChaining' broken with arrays

### ISSUE THREAD

**Issue by BlueRaja** [ember-cli/eslint-plugin-ember#997]
## Repro steps:
With
```
rules: {
  'ember/no-get': ['error', { useOptionalChaining: true }],
};
```
try to `--fix` the following code:
```
myFunc(myParam) {
  get(myParam, '0.myProperty');
}
```

## Expected results:
Probably something like
```
myParam['0']?.myProperty;
```

## Actual results:
```
myParam.0?.myProperty;
```
which crashes because `0` is not a valid identifier name

**Comment by bmish** [comment]
I tried your example and the resulting autofix looks correct.

```js
myObject.myArray?.[0]?.myProperty;
```

Are you sure you tested with the latest version of this plugin?

**Comment by BlueRaja** [comment]
Hmm, seems like I oversimplified the example.  Try this.

I'm running `eslint 7.12.1`, `eslint-plugin-ember 9.4.0`

**Comment by bmish** [comment]
@BlueRaja thanks, I opened a fix in #998.
